### DISCUSSION TITLE
MIME Type

### DISCUSSION THREAD

**Discussion by mbarkhau** [cuelang/cue#921] [Q&A]
Is there any recommendation for a MIME type to use?

Perhaps `application/cuelang` ?

**Comment by myitcv** [comment]
AFAIU `application/cuelang` would imply a [standards-agreed MIME type](https://en.wikipedia.org/wiki/Media_type), which CUE is not to my knowledge.

So it might need to be `application/x-cuelang` for now?

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/921.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
